src/account.rs

```diff
@@ -431,6 +464,61 @@ fn public_key_cli(wallet_path: &Path, account_ix: usize) -> Result<()> {
     Ok(())
 }
 
+/// Transfers assets from account at src_account_ix to a target address.
+pub(crate) async fn transfer_cli(
+    wallet_path: &Path,
+    acc_ix: usize,
+    transfer: Transfer,
+) -> Result<()> {
+    println!(
+        "Preparing to transfer:\n  Amount: {}\n  Asset ID: 0x{}\n  To: {}\n",
+        transfer.amount, transfer.asset_id, transfer.to
+    );
+    let prompt =
+        format!("Please enter your password to unlock account {acc_ix} and to initiate transfer: ");
+    let password = rpassword::prompt_password(prompt)?;
+    let mut account = derive_account_unlocked(wallet_path, acc_ix, &password)?;
+    let provider = fuels_signers::provider::Provider::connect(&transfer.node_url).await?;
+    account.set_provider(provider);
+    let to_addr = transfer.to.parse::<Bech32Address>()?;
+    println!("Transferring...");
+
+    let (tx_id, receipts) = account
+        .transfer(
+            &to_addr,
+            1,
+            Default::default(),
+            TxParameters::new(
+                Some(transfer.gas_price),
+                Some(transfer.gas_limit),
+                Some(transfer.maturity),
+            ),
+        )
+        .await?;
+
+    let beta_3_url = crate::network::BETA_3.parse::<Url>().unwrap();
+    let block_explorer_url = match transfer.node_url.host_str() {
+        host if host == beta_3_url.host_str() => {
+            "https://fuellabs.github.io/block-explorer-v2/beta-3"
+        }
+        _ => "",
+    };
+
+    println!(
+        "\nTransfer complete!\nSummary:\n  Transaction ID: 0x{tx_id}\n  Receipts: {:#?}",
+        receipts
+    );
+    if !block_explorer_url.is_empty() {
```

Comment:
This is something extra but I thought it'd be convenient to just give the link to the transaction on the block explorer for beta-3

Reply:
I think it's a great idea! Maybe we could include it as a part of the summary rather than on a separate line? E.g.
```console
  Explorer: <url>
```